packages/web/stores/user-settings/avatar.tsx

```diff
@@ -0,0 +1,34 @@
+import { action, makeObservable, observable, computed } from "mobx";
+import { UserSetting } from ".";
+
+type Avatar = "wosmongton" | "ammelia";
+
+export type AvatarState = { avatar: Avatar };
```

Comment:
Question: considering the future with NFTs etc, are we going to represent that state all within this user setting? I'm wondering if using this for profile state, which will likely grow, is a misuse of this abstraction. Should we create a new UI config store in root store? It will later do the work of querying contracts and loading prefs from localstorage. Curious what your thoughts are.

Reply:
I agree that this abstraction is simplistic for our future use case. Since this is very lean, we can remove it easily when implementing those features. I want to avoid adding a more complex abstraction for future use cases. What do you think?

Reply:
@JoseRFelix think we should create a new store for the user's profile?

Reply:
Yes, just noticed the other issue related to the empty div. I'll create another store

Reply:
@jonator Added a commit with a profile store. Made it as simple as possible.